pkg/execution/state/redis_state/queue.go

```diff
@@ -1441,6 +1442,41 @@ func (q *queue) SetFunctionPaused(ctx context.Context, accountId uuid.UUID, fnID
 	return nil
 }
 
+// SetFunctionPaused sets the "Paused" flag (represented in JSON as "off") for the given
+// function ID's queue partition.
+// If a function is unpaused, we requeue the partition with a score of "now" to ensure that it is processed.
```

Comment:
what does this comment have to do with the function?

Reply:
nothing, thanks for catching 🙃